lib/pronto/config.rb

```diff
@@ -11,6 +11,10 @@ def initialize(config_hash = ConfigFile.new.to_h)
       end
     end
 
+    def consolidate_comments?
+      !@config_hash['consolidate_comments'].nil?
```

Comment:
@jhass correct, ENVs are not currently supported for: consolidate_comments, verbose, max_warnings, or exclusions.